### DISCUSSION TITLE
Deployed web app isnt responding to clicks but works locally

### DISCUSSION THREAD

**Discussion by LizaFeng** [vercel/community#6708] [Help] [answered]
### Summary

On local host, the application works fine but when I select the deployed link, no matter what I press, it doesnt even navigate to a page that I created. The errors that show in console is that there is no POSTGRE_URL but that error doesnt show up in vercel and I tried to create the environment variable for that but it was overridden by the project because it exist. 

### Example

https://track-job-applications.vercel.app/

### Steps to Reproduce

Opening up the browser console will show the error

**Comment by versecafe** [comment]
@LizaFeng If it's showing in the browser console that means you're trying to access the PG URL on the client side which should fail, or for some reason your logs are spilling over to the client which is odd, could you share some information on the framework you're using, and either share the code base or a minimal overview of it, also provide any error logs you can.

**Reply by LizaFeng** [reply]
Hello, I am using NextJS and the error that I got in my console from the deployed site is:
Database Error: VercelPostgresError: VercelPostgresError - 'missing_connection_string': You did not supply a 'connectionString' and no 'POSTGRES_URL' env var was found.
    at c (465-ecc81b9371e3acbd.js:6:54949)
    at Object.apply (465-ecc81b9371e3acbd.js:6:56081)
    at m (page-306c7d6b1a66f8a5.js:1:6507)
    at j (page-306c7d6b1a66f8a5.js:1:7221)
    at rk (fd9d1056-534a3af521b04580.js:1:40367)
    at iB (fd9d1056-534a3af521b04580.js:1:116376)
    at o4 (fd9d1056-534a3af521b04580.js:1:94629)
    at fd9d1056-534a3af521b04580.js:1:90692
    at fd9d1056-534a3af521b04580.js:1:90699
    at oj (fd9d1056-534a3af521b04580.js:1:90804).

**Reply by versecafe** [reply]
@LizaFeng where have you set your env var on Vercel?

**Reply by LizaFeng** [reply]
In project settings has a section with environmental variables but when I try to add the ones in my local env file, it is overridden by my project

**Reply by versecafe** [reply]
you should add your env vars at `https://vercel.com/[team]/[project]/settings/environment-variables` under the same name you normally would, and make sure you're not accidentally committing a `.env` of any form

**Reply by LizaFeng** [reply]
yep, that is where the env variables are and they were added automatically by Vercel

**Reply by LizaFeng** [reply, marked as answer]
Hello, so I found the solution through searching the general problem online and it led me to this stack overflow: https://stackoverflow.com/questions/50739869/web-app-is-not-loading-after-deploying. And I thought, oh maybe its the run command and under project settings in vercel, I override the development command with the one that I use in vscode and now it works!
